<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="Netive_App.Contacts" tests="1" time="7.669" ignored="0" failures="0" timestamp="07 Dec 2016 12:01:11 GMT" skipped="0" hostname="DESKTOP-35P1KSM" errors="1">
  <testcase name="addcontactTest" time="7.669" classname="Netive_App.Contacts">
    <error type="org.openqa.selenium.WebDriverException" message="An unknown server-side error occurred while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 40 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;DESKTOP-35P1KSM&#039;, ip: &#039;10.10.13.83&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_101&#039;
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities [{appPackage=com.android.contacts, networkConnectionEnabled=true, noReset=true, warnings={}, databaseEnabled=false, deviceName=MBONSKQSY9JNSKFE, platform=LINUX, appActivity=com.android.contacts.DialtactsContactsEntryActivity, desired={appPackage=com.android.contacts, appActivity=com.android.contacts.DialtactsContactsEntryActivity, noReset=true, platformVersion=6.0.1, platformName=Android, deviceName=MBONSKQSY9JNSKFE}, platformVersion=6.0, webStorageEnabled=false, locationContextEnabled=false, browserName=Android, takesScreenshot=true, javascriptEnabled=true, platformName=Android}]
Session ID: 44808151-b764-4f5d-b65e-50af7014e545">
      <![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 40 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'DESKTOP-35P1KSM', ip: '10.10.13.83', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_101'
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities [{appPackage=com.android.contacts, networkConnectionEnabled=true, noReset=true, warnings={}, databaseEnabled=false, deviceName=MBONSKQSY9JNSKFE, platform=LINUX, appActivity=com.android.contacts.DialtactsContactsEntryActivity, desired={appPackage=com.android.contacts, appActivity=com.android.contacts.DialtactsContactsEntryActivity, noReset=true, platformVersion=6.0.1, platformName=Android, deviceName=MBONSKQSY9JNSKFE}, platformVersion=6.0, webStorageEnabled=false, locationContextEnabled=false, browserName=Android, takesScreenshot=true, javascriptEnabled=true, platformName=Android}]
Session ID: 44808151-b764-4f5d-b65e-50af7014e545
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:40)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at io.appium.java_client.AppiumDriver.performTouchAction(AppiumDriver.java:336)
	at io.appium.java_client.TouchAction.perform(TouchAction.java:323)
	at io.appium.java_client.AppiumDriver.doSwipe(AppiumDriver.java:383)
	at io.appium.java_client.android.AndroidDriver.swipe(AndroidDriver.java:186)
	at Netive_App.Contacts.addcontactTest(Contacts.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- addcontactTest -->
</testsuite> <!-- Netive_App.Contacts -->
